modules/global-classes/api.php

```diff
@@ -0,0 +1,204 @@
+		}
+
+		if ( null === $one ) {
+			return new \WP_Error( 'entity_not_found', 'Global class not found', [ 'status' => 404 ] );
+		}
+
+		try {
+			$updated = $this->repository->patch( $id, $updated );
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Updating global class failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		return new \WP_REST_Response( $updated, 200 );
+	}
+
+	private function create( \WP_REST_Request $request ) {
+		$class = $request->get_params();
+
+		try {
+			$new = $this->repository->create( $class );
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Creating global class failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		return new \WP_REST_Response( $new, 201 );
+	}
+
+	private function arrange( $request ) {
+		$order = $request->get_params();
+
+		try {
+			$all = $this->repository->all();
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Arranging global classes failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		$differences = array_merge( $all->get_order()->diff( $order ), array_diff( $order, $all->get_order()->all() ) );
+
+		if ( ! empty( $differences ) ) {
+			return new \WP_Error( 'invalid_content', 'Global classes order invalid', [ 'status' => 422 ] );
+		}
+
+		try {
+			$updated = $this->repository->arrange( $order );
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Arranging global classes failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		return new \WP_REST_Response( $updated, 200 );
+	}
+
+	// TODO: Add sanitization when implemented on prop types [EDS-574]
+	private function register_routes() {
+		register_rest_route( Module::API_NAMESPACE, '/' . Module::API_BASE, [
+			[
+				'methods' => \WP_REST_Server::READABLE,
+				'callback' => fn() => $this->all(),
+				'permission_callback' => fn() => current_user_can( 'manage_options' ),
+			],
+		] );
+
+		register_rest_route( Module::API_NAMESPACE, '/' . Module::API_BASE . '/(?P<id>[\w-]+)', [
+			[
+				'methods' => \WP_REST_Server::READABLE,
+				'callback' => fn( $request ) => $this->get( $request ),
+				'args' => [
+					'id' => [
+						'type' => 'string',
+						'required' => true,
+					],
+				],
+				'permission_callback' => fn() => current_user_can( 'manage_options' ),
+			],
+		] );
+
+		register_rest_route( Module::API_NAMESPACE, '/' . Module::API_BASE . '/(?P<id>[\w-]+)', [
+			[
+				'methods' => \WP_REST_Server::DELETABLE,
+				'callback' => fn( $request ) => $this->delete( $request ),
+				'args' => [
+					'id' => [
+						'type' => 'string',
+						'required' => true,
+					],
+				],
+				'permission_callback' => fn() => current_user_can( 'manage_options' ),
+			],
+		] );
+
+		register_rest_route( Module::API_NAMESPACE, '/' . Module::API_BASE . '/(?P<id>[\w-]+)', [
+			[
+				'methods' => \WP_REST_Server::EDITABLE,
```

Comment:
I think it will be better to just write `PUT` see:
https://github.com/WordPress/wordpress-develop/blob/7b294142e2b0db42a0bd37b81886ce7dd7965c74/src/wp-includes/rest-api/class-wp-rest-server.php#L40

Reply:
Shouldn't we use wordpress format?